### PR TITLE
Replace Get-WmiObject and Get-CimInstance calls

### PR DESCRIPTION
**Pull request by mateimicu**
Because we support Windows 2008 and Nano Server we need to choose
between Common Information Model (Cim) and Windows Management
Instrumentation (Wmi) based on the OS type.

Signed-off-by: Micu Matei-Marius mmicu@cloudbasesolutions.com

### CODE REVIEW

**Review by ader1990 — COMMENTED** [review]
this will definitely not work with 2008, but this will be subject for another effort

**Comment by mateimicu** [comment]
@ader1990 Agree, this is just one step, that fixes almost all `Get-CimInstace` and `Get-WmiObject` calls, this is needed by Windows 2008 that supports only `Wmi` and by Nano that support only `Cim`. This patch was made with Windows Nano in mind the fact that we had mixed `wmi` and `cim` calls was blocking testing on both OS`s.

File: argus/action_manager/windows.py [hunk 1]
```diff
@@ -451,13 +453,21 @@ def _get_major_version(client):
...
         A Windows Client.
+    :param  major_version:
+        Windows Major version according to
+        https://msdn.microsoft.com/en-us/library/aa394239(v=vs.85).aspx
     """
-    cmd = r"(Get-CimInstance -Class Win32_OperatingSystem).producttype"
+    # NOTE(mmicu): For Windows 10, Windows Server 2016 and Windows Nano
+    # we use Common Information Model (Cim) and for the others we use
+    # Windows Management Instrumentation (Wmi)
+    cmdlet = "Get-CimInstance" if major_version == 10 else "Get-WmiObject"
```
Comment (ader1990) [inline argus/action_manager/windows.py]: you already have the constants defined

**Review by ader1990 — CHANGES_REQUESTED** [review]
use the already defined constants

**Review by alexcoman — APPROVED** [review]
LGTM

**Pull request merged**